lib/core/src/Ouroboros/Network/Client/Wallet.hs

```diff
@@ -475,62 +475,109 @@ type LocalStateQueryResult state = Either AcquireFailure state
 --        │       ┌───────────┴───┐             ┌────────┴───────┐
 --        └───────┤   Acquired    │────────────>│   Querying     │
 --                └───────────────┘             └────────────────┘
+-- FIXME: I think the diagram is wrong. It is possible to query multiple times
+-- while we're acquired.
 --
+-- NOTE: Using AnyCardanoEra arguably goes against the grain of the abstract
+-- block type. We might be able to use the (Header block) as replacemet for
+-- (AnyCardanoEra, Tip block).
```

Comment:
I think this is a `FIXME`, but the one above is `NOTE` :)

Reply:
I fixed the state diagram.